modules/040-node-manager/templates/node-group/_bootstrap.tpl

```diff
@@ -55,39 +41,153 @@ EOF
 
 
 function get_phase2() {
-  check_python
-  bootstrap_bundle_name="${BUNDLE}.{{ $ng.name }}"
+  bootstrap_ng_name="common.{{ $ng.name }}"
   token="$(<${BOOTSTRAP_DIR}/bootstrap-token)"
   while true; do
     for server in {{ $context.Values.nodeManager.internal.clusterMasterAddresses | join " " }}; do
-      url="https://${server}/apis/bashible.deckhouse.io/v1alpha1/bootstrap/${bootstrap_bundle_name}"
+      url="https://${server}/apis/bashible.deckhouse.io/v1alpha1/bootstrap/${bootstrap_ng_name}"
       if eval "${python_binary}" - "${url}" "${token}" <<< "$(load_phase2_script)"; then
         return 0
       fi
-      >&2 echo "failed to get bootstrap ${bootstrap_bundle_name} from $url"
+      >&2 echo "failed to get bootstrap ${bootstrap_ng_name} from $url"
     done
```

Comment:
This function is incorrect. If first attempt is failed, function fails instead of try next server

Reply:
Why? When I check this script, everything works fine:
```
# cat /tmp/1.sh
set -Eeuo pipefail
function load_phase2_script() {
  cat - <<EOF
import sys
import json
import ssl

try:
    from urllib.request import urlopen, Request
except ImportError as e:
    from urllib2 import urlopen, Request

ssl.match_hostname = lambda cert, hostname: True
request = Request(sys.argv[1])
response = urlopen(request, timeout=1)
print(response)
EOF
}

set -x
while true; do
  for url in http://192.0.0.0 https://deckhouse.ru; do
    echo $url
    if eval "python3" - "${url}" <<< "$(load_phase2_script)"; then
      echo "succes with $url"
      exit 0
    fi
    >&2 echo "failed on $url"
  done
  sleep 10
done
```

Reply:
Sorry, it's my mistake.